backend/app/usecase/repository/userurl_fake.go

```diff
@@ -6,14 +6,32 @@ var _ UserURLRelation = (*UserURLRelationFake)(nil)
 
 // UserURLRelationFake represents in memory implementation of User-URL relationship accessor.
 type UserURLRelationFake struct {
+	userURLRelations map[string]string
```

Comment:
We need to use slice here. It's many to many relationship.

Reply:
Mark as Nit for now. Discussed offline. Will have further discussion about this in the future.